### PR TITLE
Feature/#280 update theme api

### PR DESCRIPTION
**Pull request by Rahul139-stack**
Please review and approve it

### CODE REVIEW

File: src/store/Modules/Projects.ts [hunk 1]
```diff
@@ -153,6 +152,53 @@ export default class Projects extends VuexModule {
...
+    this.data.list = projects
+  }
+  // get getTheme() {
+  //   return (projectIdx) => {
+  //     return this.data.list[projectIdx].themes.list[0]
+  //   }
+  // }
+  @Action
+  async setThemeData(themeData) {
+    console.log("data", themeData)
```
Comment (ankitvats1) [inline src/store/Modules/Projects.ts:170]: Remove comments.

**Review by ankitvats1 — CHANGES_REQUESTED** [review]

File: src/components/Project/ContentTab/Tabs/Theme.vue [hunk 2]
```diff
@@ -80,7 +84,20 @@
       },
     },
     methods: {
+      handleInputCode(editor) {
+        // console.log("project-tab", this.tab)
```
Comment (ankitvats1) [inline src/components/Project/ContentTab/Tabs/Theme.vue:88]: Do not comment. Remove these.

File: src/store/Modules/Projects.ts (hunk 1 above)
Comment (Rahul139-stack) [reply src/store/Modules/Projects.ts]: Removed consoles

File: src/components/Project/ContentTab/Tabs/Theme.vue (hunk 2 above)
Comment (Rahul139-stack) [reply src/components/Project/ContentTab/Tabs/Theme.vue]: Done

File: src/store/Modules/Projects.ts [hunk 3]
```diff
@@ -153,6 +152,50 @@ export default class Projects extends VuexModule {
...
+  async setThemeData(themeData: string) {
+    const themeDataParsed = JSON.parse(themeData)
+    const { code, parentIdx, selectedtheme } = themeDataParsed
+    // const themesGetters = this.context.getters["getTheme"]
+    // const themes = themesGetters(parentIdx)
+    const payload = {
+      projectid: themeDataParsed.parent,
+      scope: "string",
+      label: selectedtheme.label,
+      type: "string",
```
Comment (ankitvats1) [inline src/store/Modules/Projects.ts:178]: type must be 'theme'

**Pull request closed without merging**